### PR TITLE
allow pipe cleaner to be crafted using any sort of wire

### PR DESCRIPTION
**Pull request by misterprimus**
#### Summary
None

#### Purpose of change

Adjusts pipe cleaner recipe to allow for the use of any types of wire
#### Describe the solution

Modifies the recipe to include all the types of wire (mild, tempered, etc...) as for this tool the type likely does not matter.

### CODE REVIEW

File: data/mods/innawood/recipes/recipe_others.json [hunk 1]
```diff
@@ -64,7 +64,17 @@
     "time": "2 m",
     "reversible": true,
```
Comment (Maleclypse) [inline data/mods/innawood/recipes/recipe_others.json:65]:
```suggestion
```

File: data/mods/innawood/recipes/recipe_others.json [hunk 2]
```diff
@@ -64,7 +64,17 @@
...
+    "components": [
+      [
+        [ "wire", 2 ],
+        [ "lc_wire", 2 ],
+        [ "mc_wire", 2 ],
+        [ "hc_wire", 2 ],
+        [ "qt_wire", 2 ],
+        [ "ch_wire", 2 ],
+        [ "cable", 2 ]
+      ]
```
Comment (Maleclypse) [inline data/mods/innawood/recipes/recipe_others.json:76]:
```suggestion
      ],
      [ [ "yarn", 1 ] ] 
```

File: data/mods/innawood/recipes/recipe_others.json [hunk 3]
```diff
@@ -62,9 +62,19 @@
...
+      [
+        [ "wire", 2 ],
+        [ "lc_wire", 2 ],
+        [ "mc_wire", 2 ],
+        [ "hc_wire", 2 ],
+        [ "qt_wire", 2 ],
+        [ "ch_wire", 2 ],
+        [ "cable", 2 ]
+      ],
+      [ [ "yarn", 1 ],[ "cotton_ball", 1 ], [ "wool_staple, 1 ] ]
```
Comment (Maleclypse) [inline data/mods/innawood/recipes/recipe_others.json:76]:
```suggestion
      [ [ "yarn", 1 ],[ "cotton_ball", 1 ], [ "wool_staple', 1 ] ] 
```

File: data/mods/innawood/recipes/recipe_others.json [hunk 4]
```diff
@@ -62,9 +62,19 @@
...
+      [
+        [ "wire", 2 ],
+        [ "lc_wire", 2 ],
+        [ "mc_wire", 2 ],
+        [ "hc_wire", 2 ],
+        [ "qt_wire", 2 ],
+        [ "ch_wire", 2 ],
+        [ "cable", 2 ]
+      ],
+      [ [ "yarn", 1 ],[ "cotton_ball", 1 ], [ "wool_staple", 1 ] ]
```
Comment (Maleclypse) [inline data/mods/innawood/recipes/recipe_others.json:76]:
```suggestion
      [ [ "yarn", 1 ], [ "cotton_ball", 1 ], [ "wool_staple", 1 ] ] 
```

**Comment by misterprimus** [comment]
Anything I need to do? I guess there were conflicting changes unrelated to what this is doing?

**Comment by Maleclypse** [comment]
> Anything I need to do? I guess there were conflicting changes unrelated to what this is doing?

It made me realize we were missing a piece in the recipe to make a pipe cleaner so I fixed that here and in base game.  You are good.

**Comment by misterprimus** [comment]
> > Anything I need to do? I guess there were conflicting changes unrelated to what this is doing?
> 
> It made me realize we were missing a piece in the recipe to make a pipe cleaner so I fixed that here and in base game. You are good.

Doesn't the act of cleaning already use a piece of cloth? Why would a pipe cleaner need an integrated piece of cloth/wool?

**Comment by Maleclypse** [comment]
> > > Anything I need to do? I guess there were conflicting changes unrelated to what this is doing?
> > 
> > 
> > It made me realize we were missing a piece in the recipe to make a pipe cleaner so I fixed that here and in base game. You are good.
> 
> Doesn't the act of cleaning already use a piece of cloth? Why would a pipe cleaner need an integrated piece of cloth/wool?

Real simple.  This is a word/action confusion.  A random piece of wire isn't a pipe cleaner.  It's only a pipe cleaner with the integrated cloth/wool.  Now, if someone wants to change the action of cleaning to use either a pre-made pipe cleaner or any random wire plus a piece of cloth, go for it.  I might even do it if I find time and remember.

**Comment by misterprimus** [comment]
> > > > Anything I need to do? I guess there were conflicting changes unrelated to what this is doing?
> > > 
> > > 
> > > It made me realize we were missing a piece in the recipe to make a pipe cleaner so I fixed that here and in base game. You are good.
> > 
> > 
> > Doesn't the act of cleaning already use a piece of cloth? Why would a pipe cleaner need an integrated piece of cloth/wool?
> 
> Real simple. This is a word/action confusion. A random piece of wire isn't a pipe cleaner. It's only a pipe cleaner with the integrated cloth/wool. Now, if someone wants to change the action of cleaning to use either a pre-made pipe cleaner or any random wire plus a piece of cloth, go for it. I might even do it if I find time and remember.

I see. I thought it was already the case that it used a piece during the mend process. Was that a recent change? I could have sworn I needed to carry cleaning supplies (and not just tools) with me for unfouling my BP guns just a few months ago.

**Pull request merged**